.github/workflows/zebra.yml

```diff
@@ -61,11 +61,10 @@ jobs:
           chmod +x ./zebrad/zebrad
       - name: Create ~/.ziggurat/config.toml
         run: |
-          git clone https://github.com/ZcashFoundation/zebra
```

Comment:
I think this should work without cloning this here. I don't see we refer to this cloned path anywhere in our code or in this `.yml` script

Reply:
Even the start command doesn't use `cargo`:
`echo start_command = \"./zebrad start\" >> ~/.ziggurat/config.toml`